nilearn/datasets.py

```diff
@@ -787,7 +787,12 @@ def fetch_craddock_2011_atlas(data_dir=None, url=None, resume=True, verbose=1):
     sub_files = _fetch_files(data_dir, filenames, resume=resume,
                              verbose=verbose)
 
-    params = dict(zip(keys, sub_files))
+    with open(os.path.join(module_path, 'description', 'craddock.rst'))\
```

Comment:
A good practice would be to use "<dataset_name>.rst" as name (here craddock_2011) and create a function to get directly the rst content.